### PR TITLE
Define basic functions on interfaces

### PR DESCRIPTION
**Pull request by slashrsm**

### CODE REVIEW

File: src/EntityBrowserDisplayInterface.php [hunk 1]
```diff
@@ -22,4 +22,25 @@
    */
   public function label();
 
+  /**
+   * Display entity browser.
```
Comment (cs-shadow) [inline src/EntityBrowserDisplayInterface.php]: Nitpick: Displays

File: src/EntityBrowserWidgetSelectorInterface.php [hunk 2]
```diff
@@ -22,4 +22,19 @@
...
+   *
+   * @return array
+   *   Form structure.
+   */
+  public function getForm();
+
+  /**
+   * Returns ID of the widget that is currently selected.
+   *
+   * @return mixed
```
Comment (cs-shadow) [inline src/EntityBrowserWidgetSelectorInterface.php]: Shouldn't this be string?

Comment (slashrsm) [inline src/EntityBrowserWidgetSelectorInterface.php]: Thanks! Fixed.

File: src/EntityBrowserDisplayInterface.php (hunk 1 above)
Comment (slashrsm) [inline src/EntityBrowserDisplayInterface.php]: Fixed.

**Comment by davereid** [comment]
Rather than getForm, I think we should use more standard methods. Maybe it would make sense to have EntityBrowser objects extend https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Form%21FormInterface.php/interface/FormInterface/8 ?

**Comment by slashrsm** [comment]
Hm... not sure how that would work for plugins. EntityBrowser could implement it, though. 

What would be more standard method name? buildForm()?

**Comment by kmoll** [comment]
I don't think we need to build the form interface into the plugins, I think that certain plugins can use a form interface if they chose, but not sure requiring is neccessary.

As far as the EntityBrowser interface.  We could implement the form interface, and the plugins (widget or selection display) would be repsonsible for populating the form with entities to be submitted by the EntityBrowser widget.  But does that limit us to a single way to implement this?

**Comment by slashrsm** [comment]
Implementing FormInterface in EntityBrowserInterface doesn't seem to limit us. We will probably need to handle submit, validation, ... one way or another so it makes sense to use what we have.

**Pull request merged**